SMV/src/test/python/testDynamicConfigAppDir.py is failing due to new max cbs retry option

```
Traceback (most recent call last):
  File "/home/travis/build/TresAmigosSD/SMV/src/test/python/testDynamicConfigAppDir.py", line 79, in test_conf_reloaded
    self.assertEqual(expected_props, current_conf)
AssertionError: {'smv[80 chars]'smv.stages': 'stage', 'smv.appId': 'PROJECT_A[27 chars]ore'} != {'smv[80 chars]'smv.maxCbsPortRetries': '10', 'smv.user_libra[58 chars]T_A'}
  {'smv.appId': 'PROJECT_A'
   'smv.appName': 'App A'
   'smv.class_dir': './target/classes'
   'smv.config.keys': ''
+  'smv.maxCbsPortRetries': '10'
   'smv.stages': 'stage'
   'smv.user_libraries': 'core'}
```

--- a/src/test/python/testDynamicConfigAppDir.py
+++ b/src/test/python/testDynamicConfigAppDir.py
@@ -69,7 +69,7 @@ class RunModuleWithDynamicConfigAppDirTest(SmvBaseTest):
 
         # contents of project-a/conf/smv-app-conf.props:
         expected_props = { "smv.class_dir": "./target/classes", "smv.appName": "App A", \
-        "smv.config.keys": "", "smv.stages": "stage", "smv.appId": "PROJECT_A", "smv.user_libraries": "core" }
+        "smv.config.keys": "", "smv.stages": "stage", "smv.appId": "PROJECT_A", "smv.user_libraries": "core", "smv.maxCbsPortRetries": "20" }
 
         #  tell the app to change dirs, which should cause the reload of conf for this project
         self.smvApp.setAppDir(self.proj_a_path)

--- a/src/test/python/testDynamicConfigAppDir/project-a/conf/smv-app-conf.props
+++ b/src/test/python/testDynamicConfigAppDir/project-a/conf/smv-app-conf.props
@@ -9,3 +9,6 @@ smv.appId = PROJECT_A
 
 # dummy user_libraries key
 smv.user_libraries = core
+
+# set maximum retries of Callback Server
+smv.maxCbsPortRetries = 20

